src/test/java/com/fasterxml/jackson/core/constraints/LargeNumberWriteTest.java

```diff
@@ -1,19 +1,13 @@
 package com.fasterxml.jackson.core.constraints;
 
-import java.io.ByteArrayInputStream;
-import java.io.ByteArrayOutputStream;
-import java.io.DataInput;
-import java.io.DataInputStream;
-import java.io.DataOutput;
-import java.io.DataOutputStream;
-import java.io.StringWriter;
+import java.io.*;
```

Comment:
Can you undo this? Where feasible, it is best to keep the explicit imports.

Reply:
I've disabled collapsing imports.

Reply:
Um, err, no: 5 or more per package, should collapse, if my memory serves me.